packages/host/app/components/operator-mode/card-preview-panel.gts

```diff
@@ -0,0 +1,206 @@
+                  icon='icon-link'
+                )
+              }}
+            />
+          </:content>
+        </BoxelDropdown>
+      </div>
+    </div>
+
+    <div class='preview-body' data-test-code-mode-card-preview-body>
+      <Preview @card={{@card}} @format={{this.previewFormat}} />
+    </div>
+
+    <div class='preview-footer' data-test-code-mode-card-preview-footer>
+      <div class='footer-buttons'>
+        <button
+          class='footer-button
+            {{if (eq this.previewFormat "isolated") "active"}}'
+          {{on 'click' (fn this.setPreviewFormat 'isolated')}}
+          data-test-preview-card-footer-button-isolated
+        >Isolated</button>
+        <button
+          class='footer-button
+            {{if (eq this.previewFormat "embedded") "active"}}'
+          {{on 'click' (fn this.setPreviewFormat 'embedded')}}
+          data-test-preview-card-footer-button-embedded
+        >
+          Embedded</button>
+        <button
+          class='footer-button {{if (eq this.previewFormat "edit") "active"}}'
+          {{on 'click' (fn this.setPreviewFormat 'edit')}}
+          data-test-preview-card-footer-button-edit
+        >Edit</button>
+      </div>
+    </div>
+
+    <style>
+      :global(:root) {
+        --code-mode-preview-footer-height: 70px;
+        --code-mode-preview-header-height: 70px;
+      }
+
+      .preview-header {
+        background: white;
+        height: var(--code-mode-preview-header-height);
+        border-top-left-radius: var(--boxel-border-radius);
+        border-top-right-radius: var(--boxel-border-radius);
+        padding: var(--boxel-sp-lg);
+        display: flex;
+      }
+
+      .header-icon > img {
+        height: 25px;
+        width: 25px;
+      }
+
+      .header-icon {
+        margin-right: var(--boxel-sp-xxs);
+      }
+
+      .preview-body {
+        height: calc(
+          100% - var(--code-mode-preview-footer-height) -
+            var(--code-mode-preview-header-height)
+        );
+        overflow: scroll;
+      }
+
+      .header-actions {
+        margin-left: auto;
+      }
+
+      .preview-body :deep(.boxel-card-container) {
+        border-radius: 0;
+        box-shadow: none;
+      }
+
+      .header-title {
+        font-weight: 600;
+        font-size: 1.2rem;
+      }
+
+      .preview-footer {
+        bottom: 0;
+        height: var(--code-mode-preview-footer-height);
+        background-color: var(--boxel-200);
+        border-bottom-left-radius: var(--boxel-border-radius);
+        border-bottom-right-radius: var(--boxel-border-radius);
+      }
+
+      .footer-buttons {
+        margin: auto;
+        display: flex;
+        gap: var(--boxel-sp-sm);
+      }
+
+      .footer-button {
+        padding: var(--boxel-sp-xxxs) var(--boxel-sp-lg);
+        font-weight: 600;
+        background: transparent;
+        color: var(--boxel-dark);
+        border-radius: 6px;
+        border-color: var(--boxel-400);
+        border-width: 1px;
```

Comment:
If you use `border: 1px solid var(--boxel-400)` this subtle bevelling will disappear:

<img width="374" alt="Boxel 2023-08-30 15-20-04" src="https://github.com/cardstack/boxel/assets/43280/c56d4a46-3d4c-49e9-9e43-00d3756fbbb5">

Reply:
Nice! Much better. Will use it